src/templates/form/modal.html

```diff
@@ -19,11 +19,11 @@ <h2 class="pf-widget-headline"></h2>
             {{country}}
             <input class="pf-field-half-width" name="referralEmail" type="text">
```

Comment:
I think this needs a `aria-labelledby`?

Reply:
Think I've decided on `aria-label` for these since they don't have a `<label>` but I will also update the custom forms which will use `aria-labelledby` 👍 Thanks for pointing that out.